test/rdoc/test_rdoc_markup_to_html_crossref.rb

```diff
@@ -15,6 +15,12 @@ def test_convert_CROSSREF
     result = @to.convert 'C1'
 
     assert_equal para("<a href=\"C1.html\"><code>C1</code></a>"), result
+
```

Comment:
When encountering a non-linkable code block, like `+FOO+`, it should be rendered as `<code>FOO</code>`, which is the current behaviour.

So there should be an assertion like:

```rb
    result = @to.convert '+FOO+'
    assert_equal para("<code>FOO</code>"), result
```

but it's failing with the current implementation.

Reply:
Fixed.